javascript/javascript3/week3/index.js

```diff
@@ -0,0 +1,165 @@
+
+    fetchScreenshotApi(refs.input.value).then(data => {
+        createOneScreenshotMarkup(data.screenshotUrl);
+        screenshotUrl = data.screenshotUrl;
+        console.log(screenshotUrl);
+        if (screenshotUrl === undefined) {
+            refs.libraryList.innerHTML = `Vi did not find any matches`;
+        }
+    }).catch(error => console.log(error))
+    
+}
+
+
+function onSaveScreenshotBtnClick() {
+    refs.input.value = "";
+    refs.libraryList.innerHTML = "Screenshot saved in library";
+    console.log(screenshotUrl);
+    fetchSavingScreenshot(screenshotUrl).then(data=> console.log(data)).catch(error=>console.log(error))
+
+}
+
+function onLibraryScreenshotsBtnClick() {
+
+    fetchLibraryScreenshots().then(data => {
+        refs.libraryList.innerHTML = "";
+        createLibraryMarkup(data)
+    }).catch(error => console.log(error))
+}
+
+
+function onDeleteBtnClick(event) {
+    if (event.target.classList.contains('delete-btn')) {
+        const id = event.target.dataset.id
+        console.log(id);
+        fetchDeleteScreenshot(id);
+        setTimeout(()=>{ fetchLibraryScreenshots().then(data => createLibraryMarkup(data))
+        .catch(error=>console.log(error))
+    }, 2000)
+       
+        
+  }
+    return
+}
+
+async function fetchScreenshotApi(website) {
+
+      const options = {
+	method: 'GET',
+	headers: {
+		'X-RapidAPI-Key': '4375eace25msh493d4bce134a3cbp1743a9jsn80069769671c',
+        'X-RapidAPI-Host': 'website-screenshot6.p.rapidapi.com',
+         'Content-Type': 'application/json'
+     }    
+    };  
+
+    
+    const response = await fetch(`${BASE_URL_SCREENSHOT_API}?RapidAPI =${KEY_API_SCREENSHOT}&url=https://${website}&width=400&height=600&fullscreen=false&Request=rapidapi.com`, options);
+    const data = await response.json();
+    console.log(data)
+    return data;
+}
+
+
+
+async function fetchSavingScreenshot(url) {
+    console.log(url)
+    
+    const savedUrl = {
+    url
+    }
+console.log(savedUrl)
+    const options = {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json; UTF-8'
+        },
+        body: JSON.stringify(savedUrl)
+    }
+    
+    const response = await fetch(`${BASE_URL_CRUD_CRUD}/${KEY_API_CRUD_CRUD}/library`, options);
+    const data = await response.json();
+    return data
+}
+
+
+async function fetchLibraryScreenshots() {
+
+     const options = {
+        method: 'GET',
+        headers: {
+            'Content-Type': 'application/json; UTF-8'
+        }
+    
+    }
+    
+    const response = await fetch(`${BASE_URL_CRUD_CRUD}/${KEY_API_CRUD_CRUD}/library`,options);
+    const data = await response.json();
+    console.log('done library fetch', data)
+    return data;
+}
+
+
+async function fetchDeleteScreenshot(id) {
+    
+    const options = {
+        method: 'DELETE',
+    }
+
+    const response = await fetch(`${BASE_URL_CRUD_CRUD}/${KEY_API_CRUD_CRUD}/library/${id}`, options);
+    
+    return ;
```

Comment:
Same as above (return statement on the last line)

Reply:
I activated Pritteer and make changes. It is ok now?